grc.fraudDetection_web/src/main/resources/schema.tql

```diff
@@ -0,0 +1,186 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# dataset without banks
+# https://www.kaggle.com/datasets/kartik2112/fraud-detection?select=fraudTrain.csv
+
+ define
+
+name sub attribute,
+    value string;
+
+first_name sub attribute,
+    value string;
+
+last_name sub attribute,
+    value string;
+
+job sub attribute,
+    value string;
+
+gender sub attribute,
+    value string,
+    regex "^(F|M|O)$";
+
+company_type sub attribute,
+    value string;
+
+timestamp sub attribute,
+    value datetime;
+
+date_of_birth sub attribute,
+    value datetime;
+
+card_number sub attribute,
+    value long;
+
+amount sub attribute,
+    value double;
+
+transaction_number sub attribute,
+    value string;
+
+street sub attribute,
+    value string;
+
+city sub attribute,
+    value string;
+
+state sub attribute,
+    value string;
+
+zip sub attribute,
+    value long;
+
+longitude sub attribute,
+    value double;
+
+latitude sub attribute,
+    value double;
+
+Geo_coordinate sub entity,
+    owns longitude,
+    owns latitude,
+    plays locate:geo,
+    plays geolocate:geo,
+    plays same_place:located_com,
+    plays same_place:located_per;
+
+Address sub entity,
+    owns street,
+    owns city,
+    owns state,
+    owns zip,
+    plays locate:location;
+
+Card sub entity,
+    owns card_number,
+    plays bank_account:attached_card,
+    plays transaction:used_card;
+
+Person sub entity,
+    owns first_name,
+    owns last_name,
+    owns gender,
+    owns job,
+    owns date_of_birth,
+    plays bank_account:owner,
+    plays locate:identify,
```

Comment:
If a person should be able to be the `identify` in `geolocate` as well, then it should be OK to just specify the supertype `geolocate`. Or is the intent to describe that a person must have an address, but a company might not?